assets/dev/js/frontend/handlers/image-carousel.js

```diff
@@ -71,7 +71,7 @@ export default class ImageCarousel extends elementorModules.frontend.handlers.Sw
 		}
 
 		if ( elementSettings.image_spacing_custom ) {
-			swiperOptions.spaceBetween = elementSettings.image_spacing_custom.size;
+			swiperOptions.spaceBetween = this.getSpaceBetween();
```

Comment:
I think this might need to be added per breakpoint also in the `swiperOptions.breakpoints` object above, no?

Reply:
@udidol Good catch. Done.